ts/editor/editor-toolbar/ImageOcclusionButton.svelte

```diff
@@ -31,29 +30,21 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         <ButtonGroupItem>
             <IconButton
                 id="io-mask-btn"
+                class={$showMaskEditor ? "active-io-btn" : ""}
                 on:click={() => {
-                    if (get(maskEditorButtonPressed)) {
-                        bridgeCommand(`toggleMaskEditor:${false}`);
-                    } else {
-                        bridgeCommand(`toggleMaskEditor:${true}`);
-                    }
-                    maskEditorButtonPressed.set(!get(maskEditorButtonPressed));
+                    $showMaskEditor = !$showMaskEditor;
+                    isShowMaskEditor.set($showMaskEditor);
```

Comment:
Do we need showMaskEditor and the set call here? Why not just directly refer to $isShowMaskEditor?

Reply:
This is called on button click and handle the switch between mask editor and note editor.

Reply:
Why not remove the alias and access/set the store directly?

```diff
diff --git a/ts/editor/NoteEditor.svelte b/ts/editor/NoteEditor.svelte
index 9ff14bcae..dd5b96237 100644
--- a/ts/editor/NoteEditor.svelte
+++ b/ts/editor/NoteEditor.svelte
@@ -390,14 +390,13 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     import ImageOcclusionPage from "image-occlusion/ImageOcclusionPage.svelte";
     import type { IOMode } from "image-occlusion/lib";
     import { exportShapesToClozeDeletions } from "image-occlusion/shapes/to-cloze";
-    import { ioMaskEditorVisibleStore } from "image-occlusion/store";
+    import { ioMaskEditorVisible } from "image-occlusion/store";
 
     import { mathjaxConfig } from "../editable/mathjax-element";
     import CollapseLabel from "./CollapseLabel.svelte";
     import * as oldEditorAdapter from "./old-editor-adapter";
 
     let isIOImageLoaded = false;
-    const ioMaskEditorVisible = ioMaskEditorVisibleStore;
     let imageOcclusionMode: IOMode | undefined;
     async function setupMaskEditor(options: { html: string; mode: IOMode }) {
         imageOcclusionMode = options.mode;
diff --git a/ts/editor/editor-toolbar/ImageOcclusionButton.svelte b/ts/editor/editor-toolbar/ImageOcclusionButton.svelte
index 550f86d1b..3b17ad121 100644
--- a/ts/editor/editor-toolbar/ImageOcclusionButton.svelte
+++ b/ts/editor/editor-toolbar/ImageOcclusionButton.svelte
@@ -6,7 +6,7 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     import ButtonGroup from "components/ButtonGroup.svelte";
     import DynamicallySlottable from "components/DynamicallySlottable.svelte";
     import IconButton from "components/IconButton.svelte";
-    import { ioMaskEditorVisibleStore } from "image-occlusion/store";
+    import { ioMaskEditorVisible } from "image-occlusion/store";
 
     import ButtonGroupItem, {
         createProps,
@@ -16,7 +16,6 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     import { mdiViewDashboard } from "./icons";
 
     export let api = {};
-    const ioMaskEditorVisible = ioMaskEditorVisibleStore;
 </script>
 
 <ButtonGroup>
@@ -32,7 +31,7 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
                 id="io-mask-btn"
                 class={$ioMaskEditorVisible ? "active-io-btn" : ""}
                 on:click={() => {
-                    ioMaskEditorVisibleStore.set(!$ioMaskEditorVisible);
+                    $ioMaskEditorVisible = !$ioMaskEditorVisible;
                 }}
             >
                 {@html mdiViewDashboard}
diff --git a/ts/image-occlusion/store.ts b/ts/image-occlusion/store.ts
index c37736d43..bf56b688f 100644
--- a/ts/image-occlusion/store.ts
+++ b/ts/image-occlusion/store.ts
@@ -10,4 +10,4 @@ export const zoomResetValue = writable(1);
 // it stores the tags for the note in note editor
 export const tagsWritable = writable([""]);
 // it stores the visibility of mask editor
-export const ioMaskEditorVisibleStore = writable(true);
+export const ioMaskEditorVisible = writable(true);
```